config/version.go

```diff
@@ -33,7 +33,7 @@ const VersionMajor = 4
 
 // VersionMinor is the Minor semantic version number (x.#.z) - changed when backwards-compatible features are introduced.
 // Not enforced until after initial public release (x > 0).
-const VersionMinor = 1
+const VersionMinor = 3
```

Comment:
Shouldn't this be 2?

Reply:
Oh I think I figured it out - In #6340 I had to manually manage the merge back and neglected to bump the version number to 2. This is okay to go from 1 -> 3.